deploy/playbooks/roles/common/templates/prod_nodes.py.j2

```diff
@@ -63,6 +63,16 @@ nodes = {
         'labels': ['amd64', 'x86_64', 'jessie', 'huge'],
         'provider': 'openstack'
     },
+    'trusty_small_unique': {
+        'script': dedent("""#!/bin/bash
+        curl -u {{ prado_user | default('admin') }}:{{ prado_token }} -L "{{ prado_url }}/setup/slave/?token={{ jenkins_prado_token }}&executors=1&labels=amd64+trusty+small+x86_64+unique&nodename=trusty_small__%s" | bash
```

Comment:
You need to add the '_unique' onto the nodename you pass to prado or mita could confuse this with another node type.